xknxproject/xml/parser.py

```diff
@@ -244,7 +244,9 @@ def load(self, language: str | None) -> None:
                 continue
             device.application_program_ref = application_program_ref
             for com_object in device.com_object_instance_refs:
-                com_object.resolve_com_object_ref_id(application_program_ref)
+                com_object.resolve_com_object_ref_id(
+                    application_program_ref, int(self.project_info.schema_version)
```

Comment:
Maybe `self.project_info.schema_version` should be an integer instead of a string.

Reply:
Totally!

Reply:
I'll leave this one to you :wink: Or I'll change it in another PR.